style.css

```diff
@@ -0,0 +1,58 @@
+.container {
+  display: flex;
+}
+
+#carousel {
+  width: 390px;
+  display: flex;
+  overflow: hidden;
+  padding: 0;
+  position: relative;
+  margin: 50px auto;
+  border: 1px solid black;
+  border-radius: 3px;
+  background-color: #C4C4C4;
+}
+
+li {
+  width: 130px;
+  height: 130px;
+  list-style: none;
+}
+
+button {
+  position: absolute;
```

Comment:
For my mind, in this case, absolute is not required for buttons, you can write CSS without it